src/main/java/com/farao_community/farao/gridcapa/data_bridge/FileTransferFlow.java

```diff
@@ -49,7 +51,7 @@ public IntegrationFlow unzipArchivesIntegrationFlow() {
         return IntegrationFlows.from("archivesChannel")
                .log(LoggingHandler.Level.INFO, PARSER.parseExpression("\"Pre-treatment of file \" + headers.file_name"))
 
-                .<File, Boolean>route(this::isZip, m -> m
+                .<File, Boolean>route(file -> doUnzip && isZip(file), m -> m
```

Comment:
I prefer that we put isZip(file) && do-unzip. I find it makes more sense if isn't zip there's no point in evaluating the value of do-unzip. Knowing that the && operator evaluates operands from left to right and for each operand, converts it to a boolean. If the result is false, stops and returns the original value of that operand. @Godelaine what do you think ?